service/mantle/training/TrainingServices.xml

```diff
@@ -0,0 +1,132 @@
+        <description>To add details of new product in respective table</description>
+        <in-parameters>
+            <parameter name="productName" required="true"/>
+            <parameter name="price" required="true" type="BigDecimal"/>
+            <parameter name="description"/>
+            <parameter name="priceUomId" default-value="USD"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="productId"/>
+
+        </out-parameters>
+        <actions>
+            <service-call name="create#mantle.product.Product" in-map="context" out-map="context"/>
+
+            <service-call name="create#mantle.product.ProductPrice" in-map="context + [productId:productId]" out-map="context"/>
+        </actions>
+    </service>
+
+    <service verb="get" noun="Product">
+        <description>To get the details of a specific</description>
+        <in-parameters>
+            <parameter name="productId" required="true"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="productId"/>
+            <parameter name="description"/>
+            <parameter name="productName"/>
+            <parameter name="price"/>
+        </out-parameters>
+        <actions>
+            <entity-find-one entity-name="mantle.product.Product" value-field="product"/>
+            <set field="productId" from="product.productId"/>
+            <set field="description" from="product.description"/>
+            <set field="productName" from="product.productName"/>
+            <entity-find entity-name="mantle.product.ProductPrice" list="productPrice">
+                <econdition field-name="productId" from="productId"/>
+            </entity-find>
+            <set field="price" from="productPrice.price[0]"/>
+            <set field="priceUomId" from="productPrice.priceUomId[0]"/>
+        </actions>
+    </service>
+
+    <service verb="get" noun="Products">
+        <description>To get details of all the products</description>
+
+        <out-parameters>
+            <parameter name="products" type="list"/>
+        </out-parameters>
+        <actions>
+        <entity-find entity-name="Product" list="productsList">
+                <select-field field-name="productId"/>
+                <select-field field-name="productName"/>
+                <select-field field-name="description"/>
+            </entity-find>
+            <entity-find entity-name="ProductPrice" list="productPriceList">
+                <select-field field-name="productId"/>
+                <select-field field-name="price"/>
+                <select-field field-name="priceUomId"/>
+            </entity-find>
+            <set field="products" from="[]"/>
+            <iterate list="productsList" entry="product">
+                <entity-find entity-name="ProductPrice" list="productsPriceList">
+                    <econdition field-name="productId" value="${product.productId}"/>
```

Comment:
You can also use from instead of value. Please check the way to use both attributes.